backend/routers/user.py

```
@@ -14,6 +25,79 @@ class ExportFormat(str, Enum):
+            file_path = Path(file_path_str)
+            try:
+                if file_path.is_file():
+                    file_path.unlink()
+            except Exception:  # noqa: BLE001
+                logger.exception("Failed to delete expired export file for job %s", job_id)
+
+        export_jobs.pop(job_id, None)
+        
+@router.get("/me")
+async def get_current_user_info(
+    current_user = Depends(get_current_user)
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)